src/ngraph/runtime/hybrid/hybrid_backend.cpp

```diff
@@ -56,46 +41,42 @@ shared_ptr<runtime::Tensor>
                                                   const Shape& shape)
 {
     auto it = m_backend_list.begin();
-    return it->second->create_tensor(element_type, shape);
+    return (*it)->create_tensor(element_type, shape);
 }
 
 shared_ptr<runtime::Tensor> runtime::hybrid::HybridBackend::create_tensor(
     const element::Type& element_type, const Shape& shape, void* memory_pointer)
 {
     auto it = m_backend_list.begin();
-    return it->second->create_tensor(element_type, shape, memory_pointer);
+    return (*it)->create_tensor(element_type, shape, memory_pointer);
 }
 
 runtime::Handle runtime::hybrid::HybridBackend::compile(shared_ptr<Function> func)
 {
     if (m_function_map.find(func) == m_function_map.end())
     {
-        vector<shared_ptr<runtime::Backend>> backend_list;
-        for (auto p : m_backend_list)
-        {
-            backend_list.push_back(p.second);
-        }
-
         // Clone function
         FunctionInstance instance;
         instance.m_function = clone_function(*func);
 
         // Run placement pass
         ngraph::pass::Manager pass_manager;
-        pass_manager.register_pass<runtime::hybrid::pass::AssignPlacement>(backend_list);
+        pass_manager.register_pass<runtime::hybrid::pass::AssignPlacement>(m_backend_list);
+        pass_manager.register_pass<runtime::hybrid::pass::FixGetOutputElement>();
+        // pass_manager.register_pass<ngraph::pass::VisualizeTree>("graph.png");
```

Comment:
should we remove the commented-out code?

Reply:
yes.